polaris-react/src/components/Button/Button.scss

```diff
@@ -45,7 +45,11 @@
 }
 
 .Content {
-  @include text-style-button;
+  font-size: var(--p-font-size-100);
+  font-weight: var(--p-font-weight-medium);
+  line-height: var(--p-font-line-height-1);
+  text-transform: initial;
+  letter-spacing: initial;
```

Comment:
These came directly from the `text-style-button` [mixin](https://github.com/Shopify/polaris/blob/3438651591579ca741b1d42f8f58141065c1ec75/documentation/guides/legacy-polaris-v8-public-api.scss#L1927-L1938) that I updated to adhere to our new UI updates. I'm guessing they were added to the mixin to prevent inheriting styles from parent components.